operators/src/engine/query_processor.rs

```diff
@@ -87,6 +88,8 @@ where
 pub trait PlotQueryProcessor: Sync + Send {
     type PlotType;
 
+    fn plot_name(&self) -> &'static str;
```

Comment:
is this the correct place to get the name? I mean, layers also have names but what does the query processor have to do with that?

Reply:
I see that it's not plot name but rather plot type. Anyhow my question remains. If the function remains it should be called plot_type

Reply:
f0638b7

Reply:
looks good but what do you think about: output_format: "json/plain" for statistics and "json/vega" for histogram?

Reply:
bf88b3d  6328dea 26e9f77

Reply:
>use vega json spec directly instead of vega lite crate

why do you do this? Seems much less typesafe as before

Reply:
First of all, for the other plot, I had to do it manually, because the crate won't let me do it, but the plot is valid.
Second, the crate provides no IDE support.
Third, I could remove `ndarray` as a dependency.
Fourth, the plot stays valid and you can check it with https://vega.github.io/editor/.

I also thought it was a good thing, but I am quite disappointed.